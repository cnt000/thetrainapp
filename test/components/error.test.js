import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Error from '../../src/components/error'

describe('Error', () => {
  const props = {
    error: 'This is and error'
  }

  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Error error={null} />, div)
  })

  it('renders without crashing with error', () => {
    const div = document.createElement('div')
    div.innerHTML = 'This is and error'
    ReactDOM.render(<Error {...props} />, div)
  })

  test('has a valid snapshot', () => {
    const component = renderer.create(<Error {...props} />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
