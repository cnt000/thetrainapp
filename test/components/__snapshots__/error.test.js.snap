// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error has a valid snapshot 1`] = `
<span
  className="error"
>
  Error: 
  This is and error
</span>
`;
